src/views/product-downloads-view/components/downloads-section/index.tsx

```diff
@@ -226,6 +226,12 @@ const DownloadsSection = ({
     <div className={s.root}>
       <Card elevation="base">
         <div className={s.cardHeader}>
+          <div className={s.versionSwitcherWrapper}>
+            <VersionContextSwitcher
```

Comment:
Note: ordering has been updated here, based on design spec, I think we want "choose version" to be first in the DOM, and then "operating system".